packages/create-turbo/src/utils/git.ts

```diff
@@ -78,12 +74,18 @@ export function tryGitInit(root: string, message: string): boolean {
 
 export function tryGitCommit(message: string): boolean {
   try {
-    execSync("git add -A", { stdio: "ignore" });
-    execSync(`git commit -m "${message}"`, {
-      stdio: "ignore",
-    });
+    gitCommit(message);
     return true;
   } catch (err) {
     return false;
   }
 }
+
+function gitCommit(message: string) {
+  execSync(
+    `git commit --author="Turbobot <turbobot@vercel.com>" -am "${message}"`,
+    {
+      stdio: "ignore",
+    }
+  );
+}
```

Comment:
Factored out into a separate method so you get either the `try` behavior, or the `boolean` behavior as you see fit.